### PR TITLE
user-guides: services: cross-link to dns admin guide

### PR DESCRIPTION
**Pull request by philips**
We need to expand the DNS admin guide further. But, link to it for now.

---

This change is [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes.github.io/1250)

### CODE REVIEW

File: docs/user-guide/services/index.md [hunk 1]
```diff
@@ -317,6 +317,10 @@ Kubernetes also supports DNS SRV (service) records for named ports.  If the
 can do a DNS SRV query for `"_http._tcp.my-service.my-ns"` to discover the port
 number for `"http"`.
 
+The [DNS Admin guide](dns-admin) has further details.
```
Comment (devin-donnelly) [inline docs/user-guide/services/index.md]: In staging, this link appears to 404.

**Comment by devin-donnelly** [comment]
@philips, do you see the staged version? It's here: http://deploy-preview-1250.kubernetes-io-vnext-staging.netlify.com/docs/user-guide/services/

I checked the link you added and it looks like it 404s. Is it because that doc hasn't been merged yet?

**Comment by devin-donnelly** [comment]
Link still 404s; also, there is now a merge conflict. @philips , can we revisit this?

**Comment by devin-donnelly** [comment]
This now has a merge conflict as well.

**Comment by jaredbhatti** [comment]
Fixed this issue in a separate commit: http://kubernetes.io/docs/user-guide/services/#dns

**Comment by philips** [comment]
THanks for taking care of this.

**Pull request closed without merging**